src/kOS/Binding/FlightControlManager.cs

```diff
@@ -201,6 +201,8 @@ public void SelectAutopilotMode(object autopilotMode)
             if (autopilotMode is Direction)
             {
                 //TODO: implment use of direction subclasses.
+                throw new Safe.Exceptions.KOSException(
```

Comment:
This might as well throw an exception if the type is not `string` or `StringValue` as well, since setting it to `0` is illegal as well.  The exception should probably be a `KOSWrongControlValueTypeException` exception (see the `UpdateThrottle` method in the same file for an example).